### ISSUE TITLE
List of objects in class definition does not serialize

### ISSUE THREAD

**Issue by dpatel641** [josuebrunel/pysxm#23]
Hi I am new to Python: I have the code

```
class TheSystem(ComplexType):
    tagname='SYSTEM'
    def __init__(self) -> None:
        self.NAME = None
        self.PROPERTIES: list[Property] = None

class Property(ComplexType):
    tagname='PROPERTY'
    def __init__(self) -> None:
        self.NAME = None

```

The XML output I expect when I build up an object of this class and attempt to print it is:

```
<SYSTEM>
  <NAME> Example Name </NAME>
  <PROPERTIES>
    <PROPERTY>
       <NAME>Property 1</NAME>
    </PROPERTY>
     <PROPERTY>
       <NAME>Property 2</NAME>
    </PROPERTY>
     <PROPERTY>
      <NAME>Property 3</NAME>
    </PROPERTY>
      etc..
  </PROPERTIES>
</SYSTEM>
```

How can I achieve this? 

It works If I change list[Property] to Property (self.PROPERTIES: Property = None) e.g. only assign a single property object rather than a list but then I would not see an entire list Properties that I want in my XML. Is this possible?

Currently I get the error 
```
Exception has occurred: AttributeError
'list' object has no attribute 'xml'
```

**Comment by josuebrunel** [comment]
Hi @dpatel641 just pushed the patch. I'll let you once the new version is released.

```python

from typing import List

    class Property(ComplexType):
        def __init__(self, name: str, kind: int) -> None:
            self.name = name
            self.kind = kind

    class System(ComplexType):
        def __init__(self, name: str, properties: List[Property]) -> None:
            self.name = name
            self.properties: list[Property] =  properties

>>> s = System("Main", [Property("A", 2), Property("B", 5)])
>>> print(s)
```
will give 

```xml
<system>
  <name>Main</name>
  <properties>
    <property>
      <name>A</name>
      <kind>2</kind>
    </property>
    <property>
      <name>B</name>
      <kind>5</kind>
    </property>
  </properties>
</system>
```

**Comment by josuebrunel** [comment]
I released the 1.5.0. Let me know if it fixes your issue

**Comment by dpatel641** [comment]
> I released the 1.5.0. Let me know if it fixes your issue

Thank you very much, it is working as expected now :)
